guides/integration-tool-release.md

```diff
@@ -0,0 +1,32 @@
+# Releasing an Integration Tool
+
+This guide is about releasing integration tools and has nothing to do with MeiliSearch releases that are handled during the [pre-release week](./pre-release-week.md).
+
+> ⚠️ Integration tool releases are public and are certainly among our most consulted contents with the READMEs.<br>
+> Indeed, users reading the release changelogs are mostly users wanting to upgrade their dependencies, so they might consult the release description really carefully.
+>
+> **The Integration team should keep this in mind during the release process.**
+
+### Why Doing a Release?
+
+First of all, you need to check the release would include user-oriented changes. A release is not always needed after merging PRs.<br>
+Don't do a release without any purpose, or if only the text files, tests, or CI have been updated.<br>
+
+📢 **A release should be done only when there are changes concerning the code usage.**
```

Comment:
```suggestion
📢 **A release should be done only when there are changes concerning the code.**
```

Not only code usage itself. Exaple: optimization. Usage is the same, yet should be released?

Reply:
Real performance improvement -> yes. Refacto of the code that does not impact the usage -> no.
So this is only code usage for me. WDYT?

Reply:
The "code" is a too wide term. I can change the name of an internal variable. The code change, but the usage does not.

Reply:
```suggestion
📢 **A release should be done only when there are changes concerning the package usage.**
```

Reply:
```suggestion
📢 **A release should be done only when there are changes concerning the usage of the package.**
```

Reply:
Agreed. It would make it just too vage! 👍🏽

Reply:
```suggestion
📢 **A release should be done only when there are changes concerning the usage of the library.**
```

Reply:
WDYT of my new suggestions @eskombro? 😄

Reply:
Ugh, I don't know. I don't like to call it a "package" or a "library", because I always think that this applies to other kind of tools. Yet using "tool" or "integration" is too general, unclear, and even boring :)

I kind of wish I hadn't started this discussion 😂

I'd go back to the "code usage" :)

Reply:
Got it! I let it as it is now. If Charlotte thinks this is weird, we re-discuss this! (She will read all of these guides when coming back from holidays).